spec/helpers/http_mock/template.lua

```diff
@@ -0,0 +1,232 @@
+return [[
+# if not hostname then
+#   hostname = "_"
+# end
+# if not debug.port then
+#   error("debug.port is required")
+# end
+# if not shm_size then
+#   shm_size = "20m"
+# end
+daemon on;
+# if not worker_num then
+#   worker_num = 1
+# end
```

Comment:
Why are the commented-out regions still here?  If they are useful, please put in a comment why.

Reply:
They are not comments, but the logic that implements the template rendering.
This string is then consumed by pl_template.compile.